ansible/config_sonic_basedon_testbed.yml

```diff
@@ -589,6 +604,18 @@
         ignore_errors: true
         when: tacacs_enabled_by_default is defined and tacacs_enabled_by_default|bool == true
 
+      - name: Configure TACACS with PTF TACACS server
+        become: true
+        shell: "{{ tacacs_config_cmd }}"
+        loop:
+          - config tacacs authtype login
+          - config aaa authorization tacacs+
```

Comment:
>authorization [](http://example.com/codeflow?start=23&length=13)

Is there any testbed dedicated for Mx? they are using "tacacs+ local" authorization.

Reply:
Fixed, add new step to config authorization for MX